functions_framework_builder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.1-dev
+
+- many things...
```

Comment:
Maybe just mention this adds basic support for binary cloudevent messages...?

Reply:
Will do